.circleci/config.yml

```diff
@@ -45,6 +58,49 @@ jobs:
           python3 emscripten/tests/runner.py test_hello_world
           mkdir tmp-firefox-profile/
           echo 'user_pref("javascript.options.shared_memory", true);' >> tmp-firefox-profile/user.js
+          mkdir -p .config/openbox
```

Comment:
please add a comment here, maybe "set up openbox and X config (if we move to headless mode, this may not be needed)" - ?